kasa/smartprotocol.py

```diff
@@ -130,6 +130,9 @@ async def _execute_query(self, request: Union[str, Dict], retry_count: int) -> D
             _LOGGER.isEnabledFor(logging.DEBUG) and pf(response_data),
         )
 
+        if not response_data:
+            raise SmartDeviceException("Unexpected empty response")
```

Comment:
If this is where the {} was being received maybe update the message to "Unexpected empty response, is the request valid?" or similar.

Reply:
So this happens when executing something like this:
```
kasa -d --host 192.168.xx.xx command control_child '{"device_id":"8022852468EC205xx13BC02002","requestData":{"this_should_be_method":"some_method"}}'
```
Which does this:
```
{"method":"control_child","params":{"device_id":"8022852468EC20xx02002","requestData":{"this_should_be_method":"some_method"}},"requestID":1202024579699183616,"request_time_milis":1706626592337,"terminal_uuid":"yltgxdTKly1hdfGosYoaVA=="}
DEBUG:kasa.klaptransport:192.168.xx << {}
DEBUG:kasa.smartprotocol:192.168.xx << {}
```